docs/content/en/highlights/2021-07-13-helm-customConfig.md

```diff
@@ -0,0 +1,324 @@
+---
+date: "2021-07-13"
+title: "Introduction of `customConfig` and deprecation notice for TOML based config keys"
+short: "Introduction of `customConfig`"
+description: "Configure Vector directly through your `values.yaml` without having to converting to TOML"
+authors: ["spencergilbert"]
+pr_numbers: [8079]
+release: "0.15.0"
+hide_on_release_notes: false
+badges:
+  type: "deprecation"
+  platforms: ["helm"]
+  domains: ["config"]
+---
+
+With the release of Vector 0.15.0, we have introduced a new method of configuring Vector through Helm.
+This new method uses YAML configuration files, and in a coming release will become the default configuration
+method.
+
+**You do not need to upgrade immediately. The deprecated keys will not be removed before Vector hits 1.0**
+
+## Upgrade Guide
+
+### Breaking: updated container args
+
+The argument passed to Vector has been changed to `--config-dir` which will load any TOML, YAML, or JSON
+files added to the "/etc/vector" directory.
+
+### globalOptions and logSchema
+
+Any [global options](https://vector.dev/docs/reference/configuration/global-options/) can be moved directly
```

Comment:
```suggestion
Any [global options](/docs/reference/configuration/global-options/) can be moved directly
```

cc @lucperkins on this change. We should be using relative paths, but I want to make sure there isn't some strange edge case that requires absolute.

Reply:
Agreed, relative is better.